releasenotes/README.md

```diff
@@ -26,12 +26,54 @@ Additional documentation:
+
+    * `decomposePermissionSetBeta` — Decompose the [PermissionSet](https://developer.salesforce.com/docs/atlas.en-us.api_meta.meta/api_meta/meta_permissionset.htm) metadata type.
+    * `decomposeCustomLabelsBeta` — Decompose the [CustomLabels](https://developer.salesforce.com/docs/atlas.en-us.api_meta.meta/api_meta/meta_customlabels.htm) metadata type.
+    * `decomposeWorkflowBeta` — Decompose the [WorkFlow](https://developer.salesforce.com/docs/atlas.en-us.api_meta.meta/api_meta/meta_workflow.htm) metadata type.
+    * `decomposeSharingRulesBeta` — Decompose the [SharingRules](https://developer.salesforce.com/docs/atlas.en-us.api_meta.meta/api_meta/meta_sharingrules.htm) metadata type
+
+  This command replaces the manual steps we documented in the [April 10, 2024](./README.md#2368-april-10-2024) release notes for converting your project to decompose the additional four metadata types. Because the feature is beta, the values for the `--behavior` flag include the word `Beta`.  When the feature is generally available, we'll remove the `Beta` label; the new flag value will be `--behavior decomposePermissionSet`, for example.
+
+  (plugin-deploy-retrieve PR [#1015](https://github.com/salesforcecli/plugin-deploy-retrieve/pull/1015))
+
+* NEW: We've improved the `doctor` command so it now checks whether your computer can access certain Web sites and registries required by Salesforce CLI.  Here's truncated sample output to show which URLs it checks:
+
+    ```bash
+    $ sf doctor
+    === Running all diagnostics
+
+    pass - salesforcedx plugin not installed
+    pass - no linked plugins
+    pass - [@salesforce/plugin-deploy-retrieve] sourceApiVersion matches apiVersion
+    pass - [@salesforce/plugin-trust] can ping: https://registry.npmjs.org
+    pass - [@salesforce/plugin-trust] can ping: https://registry.yarnpkg.com
+    pass - [@salesforce/plugin-trust] can ping: https://registry.npmjs.org/
+    pass - can access: https://test.salesforce.com
+    pass - can access: https://appexchange.salesforce.com/services/data
+    pass - can access: https://developer.salesforce.com/media/salesforce-cli/sf/channels/stable/sf-win32-x64-buildmanifest
+    ...
+    ```
+    (plugin-info PR [#746](https://github.com/salesforcecli/plugin-info/pull/746), plugin-trust PR [#828](https://github.com/salesforcecli/plugin-trust/pull/828))
+  
+* FIX: If you delete a Global Action in a source-tracking-enabled org, then run `project retrieve start`, Salesforce CLI now correctly handles the corresponding local source file. Previously it did nothing because it incorrectly associated the non-existent QuickActionDefinition type with a Global Action rather than the correct QuickAction.  (GitHub issue [#2829](https://github.com/forcedotcom/cli/issues/2829), source-tracking PR [#590](https://github.com/forcedotcom/source-tracking/pull/590))
```

Comment:
```suggestion
* FIX: If you delete a Global Action in a source-tracking-enabled org, then run `project retrieve start`, Salesforce CLI will delete it locally.  Previously it did nothing because the server incorrectly lists the change as `QuickActionDefinition`.  (GitHub issue [#2829](https://github.com/forcedotcom/cli/issues/2829), source-tracking PR [#590](https://github.com/forcedotcom/source-tracking/pull/590))
```